docs/contributors/getting-started.md

```diff
@@ -186,36 +186,38 @@ These logs contain information that may be relevant for regular users.
 
 ### JSON-RPC trace
 
-To see trace of incoming/outgoing JSON communication with the text editor or
-build server, create empty files in your machine cache directory.
+**Important notice**  
+Since Metals 0.10.10 version log files should be placed in `$WORKSPACE` rather than your machine cache directory.
+
+To see trace of incoming/outgoing JSON communication with the text editor or build server, create empty files in `$WORKSPACE/.metals/` directory.
 
 ```sh
 # macOS
```

Comment:
We could join MacOS and Linux cases. Maybe even remove Windows case, just mention that you can use `type nul > ` to create the empty trace file?

Reply:
I joined macOS and Linux cases, but I didn't remove a window one. I think it's better to not delete it.